2024/12/06 18:38:07 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:38:08 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:38:23 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:38:23 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:38:26 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:38:26 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:38:27 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:38:27 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:38:27 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:38:27 [error] 170#170: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:45:03 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:45:03 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:45:05 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
2024/12/06 18:45:07 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET / HTTP/1.1", upstream: "http://172.18.0.4:80/", host: "gitlab-ce.local"
2024/12/06 18:45:07 [error] 173#173: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: gitlab-ce.local, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:80/favicon.ico", host: "gitlab-ce.local", referrer: "http://gitlab-ce.local/"
